File: Easy/JavaScript/242. Valid Anagram.js
var isAnagram = function(s, t) {
    if(s.length != t.length)
        return false;
    
    let sChars = [];
    let tChars = [];

    for(let i = 0; i < s.length; i++){
        sChars[i] = s.charAt(i);
        tChars[i] = t.charAt(i);
    }
    sChars.sort();
    tChars.sort();

    return sChars.toString() == tChars.toString();
};

File: Easy/Java/242. Valid Anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        if(s.length() != t.length())
            return false;
        
        char[] sChars = new char[s.length()];
        char[] tChars = new char[t.length()];

        for(int i = 0; i < s.length(); i++){
            sChars[i] = s.charAt(i);
            tChars[i] = t.charAt(i);
        }
        Arrays.sort(sChars);
        Arrays.sort(tChars);

        return Arrays.equals(sChars, tChars);
    }
}
